deploy/create_linux_appimage.sh

```diff
@@ -87,5 +88,6 @@ chmod a+x ./appimagetool-x86_64.AppImage
 
 ./appimagetool-x86_64.AppImage ./$APP.AppDir/ ${TMPDIR}/$APP".AppImage"
 
+mkdir -p ${OUTPUT_DIR}
```

Comment:
Doesn't `mkdir` error out of the directory is already there? That will fail CI.

Reply:
Nah. `-p` is the secret weapon for that :)
```
 -p, --parents     no error if existing, make parent directories as needed
```

Reply:
Ah, sorry missed that